dcos/dcos-ui
# DCOS-55639: Quota overview - limit banner & badges

Show no limit infobox and limited badge for quota groups
 that have services which do not have quota enforced.

Closes https://jira.mesosphere.com/browse/DCOS-55639

## Testing
0. Start localhost and login.
1. Update your `Config.dev.ts`
```
uiConfiguration: {
...
  features: {
    quota: true
  }
},
  useFixtures: true,
  ...
```
2. Go to services page and open the quota tab.
3. Verify that the infobox is shown.
4. Verify that the added test makes sense.

## Trade-offs
None.

## Dependencies
~https://github.com/dcos/dcos-ui/pull/3977~

## Screenshots
![Снимка от 2019-07-17 11-23-46](https://user-images.githubusercontent.com/40791275/61359687-85c66800-a885-11e9-919b-e6605ffeb9c2.png)
